[2024-01-31T08:17:45.509+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_data_lake_dag.dataproc_job_raw_to_formatted manual__2024-01-31T08:17:27.233428+00:00 [queued]>
[2024-01-31T08:17:45.516+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_data_lake_dag.dataproc_job_raw_to_formatted manual__2024-01-31T08:17:27.233428+00:00 [queued]>
[2024-01-31T08:17:45.516+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 6
[2024-01-31T08:17:45.533+0000] {taskinstance.py:2191} INFO - Executing <Task(DataprocSubmitJobOperator): dataproc_job_raw_to_formatted> on 2024-01-31 08:17:27.233428+00:00
[2024-01-31T08:17:45.539+0000] {standard_task_runner.py:60} INFO - Started process 142253 to run task
[2024-01-31T08:17:45.545+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'manage_data_lake_dag', 'dataproc_job_raw_to_formatted', 'manual__2024-01-31T08:17:27.233428+00:00', '--job-id', '462', '--raw', '--subdir', 'DAGS_FOLDER/data_lake_dag.py', '--cfg-path', '/tmp/tmpkgo_eus5']
[2024-01-31T08:17:45.548+0000] {standard_task_runner.py:88} INFO - Job 462: Subtask dataproc_job_raw_to_formatted
[2024-01-31T08:17:45.595+0000] {task_command.py:423} INFO - Running <TaskInstance: manage_data_lake_dag.dataproc_job_raw_to_formatted manual__2024-01-31T08:17:27.233428+00:00 [running]> on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-31T08:17:45.670+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juniortemgoua0' AIRFLOW_CTX_DAG_ID='manage_data_lake_dag' AIRFLOW_CTX_TASK_ID='dataproc_job_raw_to_formatted' AIRFLOW_CTX_EXECUTION_DATE='2024-01-31T08:17:27.233428+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-31T08:17:27.233428+00:00'
[2024-01-31T08:17:45.673+0000] {dataproc.py:2317} INFO - Submitting job
[2024-01-31T08:17:45.678+0000] {connection.py:234} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-01-31T08:17:45.683+0000] {base.py:83} INFO - Using connection ID 'google_cloud_default' for task execution.
[2024-01-31T08:17:45.865+0000] {dataproc.py:2329} INFO - Job 891d7f52-c79f-4475-bc97-6d7045a4f4a4 submitted successfully.
[2024-01-31T08:17:45.877+0000] {dataproc.py:2363} INFO - Waiting for job 891d7f52-c79f-4475-bc97-6d7045a4f4a4 to complete
[2024-01-31T08:19:56.815+0000] {dataproc.py:2367} INFO - Job 891d7f52-c79f-4475-bc97-6d7045a4f4a4 completed successfully.
[2024-01-31T08:19:56.830+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=manage_data_lake_dag, task_id=dataproc_job_raw_to_formatted, execution_date=20240131T081727, start_date=20240131T081745, end_date=20240131T081956
[2024-01-31T08:19:56.867+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-01-31T08:19:56.892+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
